general/solomiia_tymoshchuk/lecture3/_3/Shape.java

```diff
@@ -0,0 +1,32 @@
+package general.solomiia_tymoshchuk.lecture3._3;
+
+import java.util.ArrayList;
+
+public interface Shape {
```

Comment:
the interface doesn't restrict a list of classes that can extend it (see the task description)